keras/tanda_keras.py

```diff
@@ -36,7 +37,7 @@ class TANDAImageDataGenerator(ImageDataGenerator):
     """
 
     def __init__(self,
-                 tan,
+                 tan_path,
```

Comment:
Definitely need to allow user to pass in TAN object. Can use `isinstance(tan, str)`

Reply:
Fixed